README.md

```diff
@@ -0,0 +1,6 @@
+# My INF 7420 Project
+
+Welcome to my web design project for INF 7420. I am building a portfolio website to showcase my work to prospective employers.
+
+# Wireframes
+Here are the wireframes that I've used for planning and designing my page.
```

Comment:
Please include the images in the README.md file

e.g.,

`![Wireframe of Homepage](wireframes/wireframe-homepage.jpg)`